### DISCUSSION TITLE
SSR Meta Data example/feature. Like React-Helmet (React-Helmet-Async)

### DISCUSSION THREAD

**Discussion by Antonio112009** [TanStack/router#487] [Ideas]
To my believe, there should be an example of dynamically changing the `<head>` of a website. People like me might be aware of how to do so on a client-side, but when it comes to SEO optimisation and preview in social media - the data shown there is the one that is included in the default `index.html` file.

It will be beneficial over other route libraries to include an example of how you can modifying on a page change the head of the HTML on a server-side from a fetched data (when it comes to blog, news and similar type pages - hardcoding would not work. Only data fetching) 

Thank you

**Comment by dannywamuya** [comment]
Have you made any progress on this? I can get pages to ssr  but I'm not able to modify `<head/>` appropriately on the server
